Package@swift-6.0.swift

```diff
@@ -123,6 +123,8 @@ extension Array where Element == PackageDescription.SwiftSetting {
       .enableUpcomingFeature("InternalImportsByDefault"),
 
       .define("SWT_TARGET_OS_APPLE", .when(platforms: [.macOS, .iOS, .macCatalyst, .watchOS, .tvOS, .visionOS])),
+
+      .define("SWT_NO_FILE_IO", .when(platforms: [.wasi])),
```

Comment:
We don't use Foundation's `FileHandle`, we use some raw C file API. Regardless, since Wasm executes in a sandboxed environment and doesn't have file system access by default, it seems we should probably _not_ try to enable this code because it won't function correctly even if the API is implemented.

Reply:
Ah, ok. It makes sense to me👍